src/account_multisig.ts

```diff
@@ -19,6 +19,13 @@ export const MULTISIG_DEPOSIT = new BN('0');
 export const MULTISIG_CHANGE_METHODS = ['add_request', 'add_request_and_confirm', 'delete_request', 'confirm'];
 export const MULTISIG_CONFIRM_METHODS = ['confirm'];
 
+// We deploy an empty contract when we disable 2FA (main.wasm), so we must check for that
+// contract's hash OR no code_hash to be sure it's safe
+const ALLOW_2FA_ENABLE_HASHES = [
+    'E8jZ1giWcVrps8PcV75ATauu6gFRkcwjNtKp7NKmipZG',
```

Comment:
@Patrick1904 most likely issue is that we have some other multisig hashes (see MULTISIG_CONTRACT_HASHES) in wallet

ideally this should be passable through config / environment variable / etc

Reply:
@vgrichina AFAIU `'11111111111111111111111111111111'` is the default and whenever we disable multisig in the wallet we deploy the same `'E8jZ1giWcVrps8PcV75ATauu6gFRkcwjNtKp7NKmipZG'` hash so we check to make sure that the account has is either of those two before deploying a multisig contract here.

Reply:
> ideally this should be passable through config / environment variable / etc
Good point. Added `process.env.ALLOW_2FA_ENABLE_HASHES` 👍

Reply:
I think it is likely safer to pass it as a part of config / options in constructor / etc

because depending on bundler you won't get environment variable substituted in libraries and it could be confusing and hard to work around

Reply:
> @vgrichina AFAIU `'11111111111111111111111111111111'` is the default and whenever we disable multisig in the wallet we deploy the same `'E8jZ1giWcVrps8PcV75ATauu6gFRkcwjNtKp7NKmipZG'` hash so we check to make sure that the account has is either of those two before deploying a multisig contract here.

we historically have deployed different multisig contract versions

Reply:
@vgrichina all the accounts that I tested had the same hash deployed after disabling the multisig. Is there any way to query for additional potential hashes?